navel.lisp

```diff
@@ -126,6 +126,36 @@
 ;;         (multiple-value-call 'format stream "~&~@{~A~#[~:; ~]~}~%" name
 ;;           (trades-profits day))))))
 
+;; this runs on (slot-reduce maker lictor trades) or CSV tradelog
+;; (let (skipped (trades (sort * '> :key 'volume)))
+;;   (loop
+;;     (multiple-value-bind (price took paid) (trades-profits trades)
+;;       (if (null price)
+;; 	  (return (values took paid trades skipped))
+;; 	  (format t "~&~A ~A ~A" price took paid))
+;;       (if (plusp (price price))
```

Comment:
the following branches of this `if` rhyme really badly; the given/taken-cross-primary/counter metapattern has occurred many times, although it's wonky enough that I'm still collecting discrete instances before attacking the metapattern.